docs/faq.md

```diff
@@ -1,12 +1,12 @@
 # FAQ (Work in Progress)
 
 ## Why Tern?
-Tern was created to help developers meet open source compliance requirements for containers. Open source software compliance is a hard problem in general but it gets harder with containers due to the ability to reuse diff filesystems. How those filesystems were created is still an ad hoc process. If you happen to have an lwn subscription you can read an article about it [here](https://lwn.net/Articles/752982/).
+Tern was created to help developers meet open source compliance requirements for containers. Open source software compliance is a hard problem in general but it gets harder with containers due to the ability to reuse diff filesystems. How those filesystems were created is still an ad hoc process. If you happen to have a lwn subscription you can read an article about it [here](https://lwn.net/Articles/752982/).
```

Comment:
While you are here, can you change `lwn` to `LWN`?

Reply:
sure :)